core/frontend/src/utils/pattern_validators.ts

```diff
@@ -0,0 +1,40 @@
+export function validate_port(port: number, public_range = true): boolean {
```

Comment:
`validate_socket_port`, we are not validating serial, usb, ethernet ports.

Reply:
True.